configs/README.md

```diff
@@ -0,0 +1,31 @@
+# Configuration files
+
+Directories `data` and `representation` contain configuration files required to reproduce results of the study.
+Directory `models` contains an exemplary configuration file and a script to generate all possible hyperparameter configurations.
+
+All configuration files are parsed by methods implemented in `graphrepr.config`.
+
+## Dataset configuration files
+
+These configuration files define preprocessing and cost function for each dataset as well as paths to files with data.
+These paths should be updated to point to correct destinations.
+
+ESOL (scaffold) and QM9 datasets are provided in the repository while the remaining datasets (including splits) are publicly available and can be obtained as described in Section "Availability of data and materials".
+
+
+## Model configuration files
+
+File `1-model.cfg` is an example of a configuration of model architecture.
+To create all possible configurations of architectures, enter `models` directory and use this command:
+
+```bash
+python make_configs.py
+```
+
+The list of the 100 architectures used in the study is the following:
```

Comment:
I think we should describe about which 100 architectures we are writing here

Reply:
We used the same config files for GCNs, D-MPNN and MAT. Would it be enough to list the names of sections describing these experiments?

Reply:
But... are these random 100 architectures? Or selected by the hyperparameter search? I think it would be good to add a short explanation how we arrived at this set of config files.

Reply:
Oh, OK. I'll add this.

Reply:
I updated the README file (commit [86d373f](https://github.com/gmum/graph-representations/pull/3/commits/86d373f47a32e9127e61ebef09cf8d7670ca5632)).